google/cloud/aiplatform/matching_engine/_protos/match_service_pb2_grpc.py

```diff
@@ -32,10 +32,10 @@
 # Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
 
 """Client and server classes corresponding to protobuf-defined services."""
-from google.cloud.aiplatform.matching_engine._protos import match_service_pb2
-
 import grpc
 
+import match_service_pb2 as match__service__pb2
```

Comment:
Not sure if this is ok. Tried to run the command from multiple directories but this is a persistent behavior

Reply:
If the command below is run in the root of the `python-aiplatform` repo, I think the import statement will be fixed
`python -m grpc_tools.protoc --proto_path=. --grpc_python_out=. google/cloud/aiplatform/matching_engine/_protos/match_service.proto`